### PR TITLE
Travis build

### LINKED ISSUE #13: set up CI
the sooner the better, especially with work being done on integration with other services.

### PR DESCRIPTION
**Pull request by ableuler**
Add automated testing and image building.
Integrate the gateway into renku helm deployment.
Closes #13

### CODE REVIEW

File: helm-chart/renku-gateway/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (leafty) [inline helm-chart/renku-gateway/LICENSE]: use a symbolic link? see renku-ui

File: helm-chart/renku-gateway/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "renku-gateway.name" -}}
```
Comment (leafty) [inline helm-chart/renku-gateway/templates/_helpers.tpl]: Use `gateway` to stay consistent with the other sub-charts ;)

```
define "gateway.name"
...
```

File: helm-chart/renku-gateway/LICENSE (hunk 1 above)
Comment (ableuler) [reply helm-chart/renku-gateway/LICENSE]: good point - done!

File: helm-chart/renku-gateway/templates/_helpers.tpl (hunk 2 above)
Comment (ableuler) [reply helm-chart/renku-gateway/templates/_helpers.tpl]: Actually, I was following the ui example (see https://github.com/SwissDataScienceCenter/renku-ui/blob/master/helm-chart/renku-ui/templates/_helpers.tpl). Should I change it there too?

Comment (leafty) [reply helm-chart/renku-gateway/templates/_helpers.tpl]: yes, make an issue, but it's not urgent ;)

**Comment by 3C111** [comment]
I ran it against the UI (gateway-integration branches) and it seemed to work fine. Did you manage to deploy it or is that not part of this PR?

**Pull request merged**